fixtures/CAPI.ts

```diff
@@ -361,6 +361,7 @@ export const CAPI: CAPIType = {
     },
     ageWarning: 'This article is over 1 year old',
     isImmersive: false,
+    isPaidContent: false,
```

Comment:
Could we just use the tag data here? What was the thinking in introducing this instead?

Reply:
Probably. Was experimenting to see if this might be a cleaner alternative to using a "pillar"